src/planeta/cooldown.js

```diff
@@ -7,14 +7,20 @@ const COOLDOWN = getConfig().PLANET_A.COOLDOWN * 1000;
 
 const getCooldownFor = passport => JSON.parse(getStoredValue("cooldown", passport) || "{}");
 
-export function addHandshake(myPassport, theirPassport) {
+export function purge(myPassport) {
```

Comment:
Me, personally, would move purging part into a different method and call this one `getCooldown` and then call purge inside of it :)

Reply:
I'd keep it like this for now.